app/Models/Factory.php

```diff
@@ -15,7 +24,12 @@ public static function createCategoryDao($username = null) {
 		}
 	}
 
-	public static function createFeedDao($username = null) {
+	/**
+	 * @return FreshRSS_FeedDAOSQLite|FreshRSS_FeedDAO
```

Comment:
```suggestion
	 * @return FreshRSS_FeedDAO
```

Returning the top parent class should be sufficient. Same for other lines below